### ISSUE TITLE
Hang when using breakpoints

### ISSUE THREAD

**Issue by moshee** [derekparker/delve#203]
Not sure if this is related to #41 or #67 (or #70). This issue seems to happen in a much more trivial case.

Delve version: git master (d9d3118)
Go version: reproduced on 1.5beta3, 1.5rc1, and 1.4.2

I can run this program normally but as soon as I add a breakpoint (through Delve or runtime.Breakpoint) it never gets anywhere, just hangs, necessitating a kill.

```
moshee@jungle(~/projects/go/src/test) ⚡ go version
go version go1.5beta3 linux/amd64
moshee@jungle(~/projects/go/src/test) ⚡ uname -smor
Linux 2.6.32-5-amd64 x86_64 GNU/Linux
moshee@jungle(~/projects/go/src/test) ⚡ cat test.go
package main

import "fmt"

func main() {
        fmt.Println("asdfasdf")
}
moshee@jungle(~/projects/go/src/test) ⚡ dlv run
Type 'help' for list of commands.
(dlv) c
asdfasdf
Process 17852 has exited with status 0
(dlv) r
Process restarted with PID 17861
(dlv) b main.main
Breakpoint 1 set at 0x400c1a for main.main ./test.go:6
(dlv) c
[program hangs using no cpu]
```

If I send ABORT, I get [this stack trace](https://gist.github.com/moshee/46a56a904a7ea8ef1cb1).

**Comment by derekparker** [comment]
I can't reproduce, however I may have a few ideas. I almost wonder if it has anything to do with the kernel version. I ran into similar issues when I first tried Travis-ci, if you don't run on their "container" Linux workers, then you get an older 2.6.x kernel - and I was experiencing issues then. That very well could be coincidence, but let me try and run in a VM with that kernel and see if I can reproduce.
